### ISSUE TITLE
problemi di connessione wifi

### ISSUE THREAD

**Issue by piteke** [Ultimaker/Cura#11583]
### Application Version

4.12.0

### Platform

osx monterey 12.2.1

### Printer

ultimaker 3

### Reproduction steps

1. procedo a preparare il file
2. mando la stampa in wifi

### Actual results

 cura perde connessione

### Expected results

stampare

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Additional information & file uploads

devo chiudere il programma e riaprire, ma non sempre funziona

[cura.log](https://github.com/Ultimaker/Cura/files/8168212/cura.log)
.

**Comment by piteke** [comment]
yes, I have been using um3 for 5 years with the software cura and the problem is not always there but sometimes yes.

**Comment by Ghostkeeper** [comment]
A couple of things from the log. It's more or less alternating between these two a bit:

```
It has been 37.99512028694153 seconds since the last response for outputdevice manual:192.168.1.63, so assume a timeout
Error transferring http://192.168.1.63/cluster-api/v1/printers - server replied: SERVICE UNAVAILABLE
```

The first one could be either the connection or the printer being broken, but the second one definitely indicates that it's the printer which is the problem here. It got a reply from the printer, but that reply is that the printer can't give the information that Cura requests. Perhaps turning the printer off and on would help, if the printing API service got stuck somewhere. It has been known to do that until a firmware update like 2 years ago. Did you update the firmware recently? The most recent version as of this writing is 5.3.0, from October 2021.

I don't think Cura can be expected to print if the printer is giving `SERVICE UNAVAILABLE` error messages. You might have to take this up with your Ultimaker's support channel instead. It's not a bug in Cura as far as I can tell.
